food.cpp

```diff
@@ -26,6 +26,15 @@ bool food::is_eaten() const noexcept {
     }
 }
 
+bool food::is_uneaten() const noexcept {
+    if (food::get_food_state() == food_state::uneaten) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+

```

Comment:
```suggestion
```
Nice that you added the complementary function for `food_state::uneaten`, but I feel it is unnecessary since strictly equivalent to `!is_eaten()`. To keep a good code coverage and limit the number of lines, I would remove it.